ga4gh/datarepo.py

```diff
@@ -23,6 +24,8 @@ def __init__(self):
         self._referenceSetIdMap = {}
         self._referenceSetNameMap = {}
         self._referenceSetIds = []
+        self._ontologyNameMap = {}
```

Comment:
I think `name` captures the functionality better, but I'm open to your suggestion for naming this variable.